app/src/test/java/ch/epfl/sdp/items/ItemBoxTest.java

```diff
@@ -30,7 +30,7 @@ public void setup() {
         PlayerManager.setCurrentUser(player);
         mockMapApi = new MockMapApi();
         Game.getInstance().setMapApi(mockMapApi);
-        Game.getInstance().setRenderer(new MockRenderer());
```

Comment:
In a future release of master branch, MockRenderer will be removed since it is redundant class. GoogleMapApi implements Renderer, so by extension MockMapApi should also implement Renderer and there was really no need for MockRenderer

Reply:
The thing is that GoogleMapsApi does not implements Renderer anymore, as the Renderer is the Activity that displays the Entities